### PR TITLE
IOS-16817: allow refresh user avatar

### PR DESCRIPTION
**Pull request by thuyBox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @thuyBox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Views/BOXUserAvatarImageView.m [hunk 1]
```diff
@@ -125,6 +125,15 @@ - (void)renderRealUserAvatarFromRemote
     }];
 }
 
+- (void)refresh
+{
+    if (self.user.modelID.length > 0) {
+        [self.request cancel];
+        self.request = [self.client userAvatarRequestWithID:self.user.modelID type:BOXAvatarTypeLarge];
+        [self renderRealUserAvatarFromRemote];
```
Comment (jlawton) [inline BoxContentSDK/BoxContentSDK/Views/BOXUserAvatarImageView.m]: How do you feel about this pattern? My gut says that having something like `[self renderRealUserAvatarFrom:request]` is better.

**Review by jlawton — APPROVED** [review]

File: BoxContentSDK/BoxContentSDK/Views/BOXUserAvatarImageView.m (hunk 1 above)
Comment (thuyBox) [reply BoxContentSDK/BoxContentSDK/Views/BOXUserAvatarImageView.m]: done

**Comment by thuyBox** [comment]
local test runs succeeded

**Pull request merged**